tests/main/invariant-tool/task.yaml

```diff
@@ -0,0 +1,29 @@
+summary: invariant-tool is detecting problems
```

Comment:
I think we should introduce a new category besides main/ for tests of helper tools (session-tool would go there too).

Reply:
Yeah, I was thinking about it on Friday. It would be extra useful IF it did not build snapd and the rest of the stack, and would just allow us to do the quick testing of the helper tools themselves.

Reply:
Something to possibly revisit in near future, not for this PR; we're growing tests for tooling, unrelated to snapd functionality, would be nice to separate them.